scripts/us_cdc/environmental_health_toxicology/OzoneCensusTractPollution.tmcf

```diff
@@ -0,0 +1,36 @@
+Node: E:OzoneCTPollution->E1
+typeOf: dcs:CensusTract
+dcid: C:OzoneCTPollution->dcid
+
+Node: E:OzoneCTPollution->E2
+observationAbout: E:OzoneCTPollution->E1
+typeOf: dcs:StatVarObservation
+observationDate: C:OzoneCTPollution->date
+variableMeasured: E:OzoneCTPollution->E3
```

Comment:
This can just be:

variableMeasured: dcid:Mean_Concentration_AirPollutant_Ozone

and you can get rid of E3

Reply:
ditto for other TMCFs too